projects/storefrontlib/src/cms-components/configurator/commons/to-configuration/to-configuration.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Input } from '@angular/core';
+import { RoutingService } from '@spartacus/core';
+
+@Component({
+  selector: 'cx-to-configuration',
+  templateUrl: './to-configuration.component.html',
+})
+export class ToConfigurationComponent {
+  @Input() productCode: string;
+  @Input() configuratorType: string;
+
+  constructor(private routingService: RoutingService) {}
+
+  toConfiguration() {
+    //later we might check for a cached configuration per product
+
+    this.routingService.go({
```

Comment:
For web crawlers that use SSR, it would be good to use `<a [routerLink]="{ cxRoute: ..., params: ... } | cxUrl">` in templates (which is transformed to normal `<a href=...>`).

Reply:
Hmhh yes. Also considered a direct link first.
I did it like this because I thought about business logic that we will later apply before navigating. But maybe we can do this also in the receiving component. So let me check this, depending on the outcome I will change the navigation.